src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component } from '@angular/core';
+import { ENTRIES } from './mock-data';
+import { Entry } from '../entry';
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.scss']
+})
+export class DashboardComponent {
+  listOfEntries = ENTRIES;
+  selectedEntry = new Entry('');
+
+  save() {
+
+    let entryAlreadyExists = false;
+
+    for (let entry of this.listOfEntries) {
```

Comment:
Instead of using a `for of` loop try using `this.listOfEntries.forEach()` or bonus points for using `this.listOfEntries.some()`